### DISCUSSION TITLE
Voronoi treemap

### DISCUSSION THREAD

**Discussion by lazarusA** [DanielVandH/DelaunayTriangulation.jl#148] [Ideas]
any plans for this? 

https://github.com/Kcnarf/d3-voronoi-treemap

**Comment by DanielVandH** [comment]
Never heard of a Voronoi treemap! While I don't have any plans for it currently, I do have plans for weighted Delaunay triangulations and power Voronoi diagrams, which are the key components of the algorithm (though the main paper references some other algorithms for this, maybe I can get around implementing those first). Maybe in a couple of months I can revisit this again? The next features I'm prioritising, though, are spatial sorting and curved boundaries, so might be a bit longer unless there's a lot more interest for it.

**Comment by lazarusA** [comment]
Indeed, they not pretty common in the science domain, but rather more in the political one. Recently, there is an increase in his use to convey messages to the general public. 

Good to know that some of the things necessary to make this work are already on the roadmap. Thanks.

**Comment by DanielVandH** [comment]
Just as an FYI, I've not completely forgotten about this, it's just that development on weighted triangulations has been a struggle. See https://github.com/JuliaGeometry/DelaunayTriangulation.jl/issues/164 if you are interested

**Comment by DanielVandH** [comment]
I commented about this on Slack but I'll also put it here just so it's also in this discussion: All the pieces for this are now in the package. The only thing left is to actually write up the algorithm.
